scipy/io/mmio.py

```diff
@@ -505,7 +532,7 @@ def _parse_body(self, stream):
         return a
 
     #---------------------------------------------------------------------------
-    def _write(self, stream, a, comment='', field=None, precision=None):
+    def _write(self, stream, a, comment='', field=None, precision=None, symmetry=None):
```

Comment:
Sure - but the question is  - what should the default be?  A new hit for performance for the non-symmetric case, or default to not-checking, and have a separate flag to ask for the check?